src/components/Requirements/RequirementGroup.vue

```diff
@@ -148,12 +148,12 @@ export default defineComponent({
       const ongoing: RequirementFulfillment[] = [];
       const completed: RequirementFulfillment[] = [];
       this.req.reqs.forEach(req => {
-        if (req.minCountFulfilled < req.minCountRequired) {
+        if (req.fulfillment.safeMinCountFulfilled < req.fulfillment.minCountRequired) {
           ongoing.push(req);
         } else if (
-          req.additionalRequirements != null &&
-          Object.values(req.additionalRequirements).some(
-            it => it.minCountFulfilled < it.minCountRequired
+          req.fulfillment.additionalRequirements != null &&
```

Comment:
Could probably break up this expression a bit / factor it into a function for better readability